app/models/target.rb

```diff
@@ -9,7 +12,18 @@ class Target < ApplicationRecord
 
   validate :max_targets
 
+  scope :target_user, ->(user) { where("user_id != ?", user.id) }
```

Comment:
The name could be more descriptive. You are quering targets from other users. maybe `not_for(user)`. So when you call it, it will be like `Target.not_for(user)` . Or `not_belong_to(user)` or something like that.

Also you can write it like `where.not(user_id: user.id)`